src/Microsoft.Health.Dicom.Api/Features/ByteCounter/ByteCountingStream.cs

```diff
@@ -0,0 +1,134 @@
+﻿// -------------------------------------------------------------------------------------------------
```

Comment:
It does seem that the ByteCounter code would belong in the healthcare-shared-components repo - seems pretty self-contained.

Reply:
Yeah that makes sense, especially since this code is originally from FHIR, so it should be sharable. Created PR https://github.com/microsoft/healthcare-shared-components/pull/314
& will remove the files here